prow/manifests/overlays/metal3/config.yaml

```diff
@@ -1711,6 +1711,11 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
+  # name: {job_prefix}-bml-integration-test-{image_os}
```

Comment:
```suggestion
  # name: {job_prefix}-bml-integration-test-{image_os}
```
Name does not match the actual keyword: one says `test`, the other says `tests`.

Reply:
done